python/pipeline/reso.py

```diff
@@ -514,10 +511,6 @@ def _make_tuples(self, key):
             self.notify({**key, 'slice': slice_id + 1}, scan) # once per slice
 
     def notify(self, key, scan):
-        # --  notification
-        import matplotlib
-        matplotlib.rcParams['backend'] = 'Agg'
-        import matplotlib.pyplot as plt
```

Comment:
Sorry, what I meant is: I couldn't find a place where `plt` is now imported. Is it still available? Did you test whether it correctly plots and sends the images?